client/src/components/new-request/submit-request/reserve-submit.vue

```diff
@@ -46,12 +46,20 @@ export default class ReserveSubmitButton extends Vue {
     if (this.setup === 'cancel') {
       return 'Stop & Send To Examination'
     }
+
+    // delete the next 3 lines when re-enabling auto and conditional approvals
+    if (this.setup !== 'assumed') {
```

Comment:
With regards to `assumed`, `consent`, `examine` etc, it might be better to store these string values in an enum. Especially if these values are used elsewhere. 
I believe if you reference an enum value in the switch Case, ie `case ApprovalTypes.CONSENT: ...`
it might also sort out the TS errors.

Reply:
the string values here are fed into the component via props and only really apply within it.
wouldn't an enum used as you suggest return a numeric index, like `ApprovalTypes.consent === 2` or whatever depending on the order.  I'm probably failing to see something obvious here but I don't get how this would help except possibly the type error.

Reply:
No, you can have an enum map to strings. It might seem pointless but it allows for strict type checking and is easier to find globally in code. Eg,
https://github.com/bcgov/business-filings-ui/blob/master/src/enums/nameRequestTypes.ts

Reply:
i got this partially implemented over the weekend and it makes no difference to the type errors.  Those strings are not used anywhere else in the app either so I think we should not bother with the enum at the moment because it doesn't seem to provide any benefit

Reply:
OK.

We should look into this in the future, and enable some of the TS strict flags. Search for "--strict" on this page: https://www.typescriptlang.org/docs/handbook/compiler-options.html